k9mail/src/main/java/com/fsck/k9/view/RecipientSelectView.java

```diff
@@ -396,6 +396,19 @@ static private boolean isPlaceholderText(String currentCompletionText) {
         return currentCompletionText.startsWith("+") && currentCompletionText.substring(1).matches("[0-9]+");
     }
 
+
+    @Override
+    public String getRecipentEmail(Recipient currenRecipient){
+        String name=currenRecipient.getDisplayNameOrAddress();
```

Comment:
No point doing the assignment here. Also please correct the spelling of 'current'.